blockchain/blockchain-wallet-v4-frontend
# Feat/bank link error logging

## Description (optional)
Adds matomo events for yodlee errors

## Testing Steps (optional)
Get the Yodlee UI or banktrasfer api endpoints to send an error, like a failed new ach account link and check matomo for `BANK_LINK_FAILED` events.
